src/input.rs

```diff
@@ -324,7 +338,23 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
             if self.ctx.mouse_mut().left_button_state != state {
                 match self.ctx.mouse_mut().left_button_state {
                     ElementState::Pressed => {
-                        self.on_mouse_press();
+                        let diff = time::get_time() - self.ctx.mouse_mut().last_click_timestamp;
+                        let within_threshold = &diff.num_milliseconds() < self.multiclick_threshold;
+                        match self.ctx.mouse_mut().click_state {
+                            ClickState::Click if within_threshold => {
+                                self.ctx.mouse_mut().click_state = ClickState::DblClick;
+                                self.on_mouse_dbl_click();
+                            },
+                            ClickState::DblClick if within_threshold => {
+                                self.ctx.mouse_mut().click_state = ClickState::TripleClick;
+                                println!("Triple Click!");
```

Comment:
Why not do something with triple click? At the very least, this should have a big `// TODO` and change the `println!` to a `debug!`.